test/language/expressions/optional-chaining/early-errors-tail-position-template-string.js

```diff
@@ -0,0 +1,21 @@
+/*---
+esid: pending
+info: |
+  template string passed to tail position of optional chain
+description: >
+  Static Semantics: Early Errors
+    OptionalChain:
+      ?.TemplateLiteral
+      OptionalChain TemplateLiteral
+features: [optional-chaining]
+negative:
+  type: SyntaxError
+  phase: parse
+---*/
+
+$DONOTEVALUATE();
+
+const a = {fn: () => {}};
```

Comment:
Non-blocking, entirely out of curiosity question: Why the arrow function here, and not just `{fn(){}}`?